apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -222,13 +221,15 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
     process.exitCode = 0;
   }
 
-  private _normalizeCwd(): void {
-    const buildFolder: string = this._heftConfiguration.buildFolder;
+  private async _normalizeCwdAsync(): Promise<void> {
```

Comment:
```suggestion
  private _normalizeCwd(): void {
```
No longer has any async operations.